conf/pyramid/sections.libsonnet

```diff
@@ -0,0 +1,267 @@
+/*
+Generic function to use for generating Pyramid/PasteDeploy INI config files. Each
+function returns an object taking the form `{section_name: section_data}` where
+`section_name` will be the name of the section in the serialized INI file and
+`section_data` is an object containing key-value pairs to list under the section. So a
+section like `{foo: {bar: baz}}` could be serialized into the following INI file:
+```
+[foo]
+bar = baz
+```
+
+Each function's arguments can change the resulting section returned. The default values
+are not particularly special, they usually represent the most common configuration
+values to make assembling the sections into complete INI files less verbose by reducing
+the need to supply overriding keyword arguments.
+
+Jsonnet requires quoting any dotted keys or keys with spaces. If you need to use
+PasteDeploy `set` keyword then enclose it in quotes (e.g. `set timeout`).
```

Comment:
Yeah this documentation is unclear, I mean quotes. The backticks are just there to indicate the text is code. I can put the code sample on its own line, hopefully then it will make more sense.